README.md

```diff
@@ -2,55 +2,57 @@
 
 # Project Name
```

Comment:
- **[OPTIONAL]** This was missed in the previous review, however I strong suggest your give your readme a proper title